Makefile

```diff
@@ -262,7 +263,7 @@ endif
 
 test/addons/.docbuildstamp: $(DOCBUILDSTAMP_PREREQS)
 	$(RM) -r test/addons/??_*/
-	$(NODE) $<
+	[ -x $(NODE) ] && $(NODE) $< || node $<
```

Comment:
Why just here? shouldn't this be handled in a global manner? Or just err with the message "to run with a precompiled node binary run `make NODE=<path_to_node> <target>`"

Reply:
Good idea